xml/templates/civicrm_data.tpl

```diff
@@ -1746,7 +1746,7 @@ VALUES
 
 -- CRM-9714
 
-SELECT @financial_type_id := max(id) FROM `civicrm_financial_type` WHERE `name` = 'Member Dues';
+SELECT @financial_type_id := max(id) FROM `civicrm_financial_type` WHERE `name` = '{ts escape="sql"}Member Dues{/ts}';
```

Comment:
@eileenmcnaughton why not instead of doing this remove this line and replace in L1752 with `@financial_type_id_md` as per https://github.com/civicrm/civicrm-core/pull/21291/files#diff-6a142c260ac3fb19d9874a52ac8b725332efcba87084fccfe15ada0f38ec3142R1694